blosc2/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
-add_cython_target(blosc2_ext blosc2_ext.pyx)
-add_library(blosc2_ext MODULE ${blosc2_ext})
+# Compile the Cython extension manually...
+add_custom_command(
+  OUTPUT blosc2_ext.c
+  COMMAND Python::Interpreter -m cython
+          "${CMAKE_CURRENT_SOURCE_DIR}/blosc2_ext.pyx" --output-file blosc2_ext.c
+  DEPENDS blosc2_ext.pyx
+  VERBATIM)
+# ...and add it to the target
+Python_add_library(blosc2_ext MODULE blosc2_ext.c WITH_SOABI)
 
 if(USE_SYSTEM_BLOSC2)
     set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
```

Comment:
```suggestion
```
This folder does not exist

Reply:
Maybe this refers to `blosc2/c-blosc2/cmake` ?  Again, I'm not specially knowledgeable about cmake :-/